plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder.java

```diff
@@ -103,7 +103,7 @@
     private boolean isBlameDisabled;
     private boolean isForensicsDisabled;
 
-    private boolean isChecksPublishingDisabled;
+    private boolean publishChecks = true; // by default, checks will be published
```

Comment:
Note that using a positive setting and `true` as default will only work for new freestyle jobs. Existing jobs will not publish the results and need to be manually reconfigured. (This is no big thing, I just want to make sure that you are aware of this issue.)

Reply:
oh, I didn't aware that, maybe it's better to make it backward compatible.

Reply:
Should be fine...

Reply:
This is not related to backward compatibility. The question is just if existing users (of Freestyle jobs!) should automatically use the GitHub checks or not. For me it is also ok that they need to manually enable.